File: src/sdk-reference/cpp/1/collection/create/snippets/create.cpp
try {
  std::string mapping = R"({
    "properties": {
      "license": { "type": "keyword" },
      "driver": {
        "properties": {
          "name": { "type": "keyword" },
          "curriculum": { "type": "text" }
        }
      }
    }
  })";

  kuzzle->collection->create("nyc-open-data", "yellow-taxi", mapping);

  std::cout << "Collection successfully created" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/create/snippets/create.cs
try {
  string mapping = @"{
    ""properties"": {
      ""license"": { ""type"": ""keyword"" },
      ""driver"": {
        ""properties"": {
          ""name"": { ""type"": ""keyword"" },
          ""curriculum"": { ""type"": ""text"" }
        }
      }
    }
  }";

  kuzzle.collection.create("nyc-open-data", "yellow-taxi", mapping);

  Console.WriteLine("Collection successfully created");
} catch  {
  Console.Error.WriteLine(e.what());
}
